Marlin/src/inc/Conditionals_adv.h

```diff
@@ -1152,3 +1152,10 @@
 #if EITHER(INPUT_SHAPING_X, INPUT_SHAPING_Y)
   #define HAS_SHAPING 1
 #endif
+
+// Toolchange Event G-code
+#if !HAS_MULTI_EXTRUDER || !(defined(EVENT_GCODE_TOOLCHANGE_T0) || defined(EVENT_GCODE_TOOLCHANGE_T1) || defined(EVENT_GCODE_TOOLCHANGE_T2) || defined(EVENT_GCODE_TOOLCHANGE_T3) || defined(EVENT_GCODE_TOOLCHANGE_T4) || defined(EVENT_GCODE_TOOLCHANGE_T5) || defined(EVENT_GCODE_TOOLCHANGE_T6) || defined(EVENT_GCODE_TOOLCHANGE_T7))
```

Comment:
Should the `defined()` checked for `EVENT_GCODE_TOOLCHANGE_T1` and onwards be `!defined()` instead? If `EVENT_GCODE_TOOLCHANGE_T1` is defined by the user as intended, wouldn't the `TC_GCODE_USE_GLOBAL_X` be disabled?

Reply:
Nevermind, I just noticed the extra parentheses `!(` around all the `defined()` calls so it makes sense.